config/schools.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    "voivodeships" => [
+        "DOLNOŚLĄSKIE",
```

Comment:
```suggestion
        "DOLNOŚLĄSKIE",
```
Create for voivodeships Enum and load here enabled values.

Reply:
done